### ISSUE TITLE
Macapype: external pipeline

### ISSUE THREAD

**Issue by davidmeunier79** [brainhackorg/global2021#131]
## Project info

**Title:**
Macapype: external pipelines

**Project lead and collaborators:**
David Meunier 
Kepkee Loh
Julien Sein
Bastien Cagna
Olivier Coulon

**Description:**
Macapype is a python package based on nipype, wrapping the tools required for NHP MRI anatomical segmentation. Macapype provide several processing pipelines, allowing optimisation and adaptation, depending on the quality of the images (SNR), acquisition sequences and antenna types, as well as species.
Today, macapype is provided as a github repo, a pip install, and a docker/singularity image. The accepted inputs have to be in BIDS format, and macapype pipelines are callable thanks to a command line interface (CLI). 

**Links :**
Macapype Github: https://github.com/Macatools/macapype
Macapype Documentation: https://macatools.github.io/macapype/index.html

**Goals for Brainhack Marseille**
ely oriented to PNH anatomical segmentation. Owever, many applications can be included starting from a good segmentation quality : 
    • ACT (anatomically contrained tractography) from diffusion data, 
    • mesh and surfaces generation
that are not pure PNH but that are useful to be linked to macapype segementation. We would like to discuss the possibility of integration o this tools as external pipelines, i.e. not directly part of macapype, but that can be added in global workflows based on nipype.

**Putative schedule :**
Monday 6th of December , 2pm (Central Europe Time): presentation of macapype, past and future functionalities
Tuesday 7th of December, 2pm (CET) : diffusion and tractography of PNH data : discussion , and strategies

Required skills :
PNH anatomical MRI processing 30-100 %
python 50-100 %
nipype 30-100 %
BIDS/BIDS derivatives 50-100 % 

**Striking [Image**]
![macapype_logo_manon](https://user-images.githubusercontent.com/7290245/143407460-dee2115d-feb6-4089-b514-f2d19deef17b.jpg)

**Comment by JuliaSprenger** [comment]
@davidmeunier79 I don't think the label `programming:R` is correct for this project, this should be `programming:python` instead.

**Comment by JuliaSprenger** [comment]
Also for the project to be considered ready to integrate you need to add a comment `Hi @Brainhack-Global/project-monitors: my project is ready!` (see issue template to submit projects)

**Comment by davidmeunier79** [comment]
Hi @Brainhack-Global/project-monitors: my project is ready!

**Comment by davidmeunier79** [comment]
> Also for the project to be considered ready to integrate you need to add a comment `Hi @Brainhack-Global/project-monitors: my project is ready!` (see issue template to submit projects)

Merci @JuliaSprenger !

**Comment by davidmeunier79** [comment]
Also I noticed the automated labelling included: programming: R, instead of programming: python

**Comment by Remi-Gau** [comment]
> Also I noticed the automated labelling included: programming: R, instead of programming: python And the tag nipype could also be added!

Yeah our automatic labeller is taking some interesting liberties sometimes. 
Will fix that

And good idea for the nipype label suggestion: will add it

**Comment by Remi-Gau** [comment]
> @Brainhack-Global/project-monitors

actrually it is supposed to be @brainhackorg/project-monitors but the typo was in the project template so totally on us. :facepalm:

**Comment by Remi-Gau** [comment]
As the project was already reviewed by the local event organizers there is no need for further review on our side. Will proceed with publication on the website.

**Comment by jhlegarreta** [comment]
It looks like this issue was not created using the project submission template: section names are different. Same applies to virtually all BHD events... The latter have been processed by the project file generation script somehow, though. Honestly. IDK how the script succeeded.

**Comment by Remi-Gau** [comment]
> The latter have been processed by the project file generation script somehow, though. Honestly. IDK how the script succeeded.

It works: I don't know why!
It doesn't work: I don't know why!

**Comment by davidmeunier79** [comment]
Many thanks to both of you @Remi-Gau and  @jhlegarreta 
One last thing: Would it be possible to add at the end of the project page: 

**Communication channels:**
https://mattermost.brainhack.org/brainhack/channels/primedre-brainhack-2021-macapype

Thanks again for your amazing job!

**Comment by Remi-Gau** [comment]
@davidmeunier79 

I am pretty sure that if you edit your top post (https://github.com/brainhackorg/global2021/issues/131#issue-1063332327) then the rest will be done almost entirely via github automation

**Comment by davidmeunier79** [comment]
> @davidmeunier79
> 
> I am pretty sure that if you edit your top post ([#131 (comment)](https://github.com/brainhackorg/global2021/issues/131#issue-1063332327)) then the rest will be done almost entirely via github automation

Yes it worked; many thanks!
